.github/workflows/main.yml

```diff
@@ -1,17 +1,58 @@
-name: iText7 CI Pipeline
+name: iText7 CI/CD Pipeline
 
 on: [push, pull_request, workflow_dispatch]
 
 jobs:
   ci:
     runs-on: windows-latest
     steps:
+      - name: Setup MSBuild
+        id: setup_msbuild
+        uses: microsoft/setup-msbuild@v1.0.2
+
+      - name: Install Ghostscript
+        id: install_ghostscript
+        shell: pwsh
+        run: |
+          choco install ghostscript -y 
+
+      - name: Install Imagemagick
+        id: install_imagemagick
+        shell: pwsh
+        run: |
+          choco install imagemagick -y
+
       - name: Checkout repository
         id: checkout_repo
         uses: actions/checkout@v2
-        
-      # Build process
-      
-      # Tests process
+        with:
+          fetch-depth: 0
+
+      - name: Clean .NET cache
+        id: clean_dotnet_cache
+        shell: pwsh
+        run: |
+          dotnet clean .\iTextCore.netstandard.sln
+          dotnet nuget locals all --clear
+
+      - name: Install NuGet packages
+        id: install_nuget_packages
+        shell: pwsh
+        run: |
+          nuget restore .\iTextCore.netstandard.sln
+
+      - name: Build solution
+        id: build_solution
+        shell: pwsh
+        run: |
+          msbuild .\iTextCore.netstandard.sln
+
+      - name: Run tests
+        id: run_tests
+        shell: pwsh
+        run: |
+          $env:ITEXT_GS_EXEC = "C:\Program Files\gs\gs9.53.3\bin\gswin64c.exe"
```

Comment:
Why do we need these environment variables to be set?

Reply:
@kulov the build process executes both testing software directly, so it uses these two variables which should be paths to .exe file as per their BUILDING.md